app/models/alert_type_rating.rb

```diff
@@ -38,6 +38,7 @@ class AlertTypeRating < ApplicationRecord
   has_many :intervention_types, through: :alert_type_rating_intervention_types
 
   scope :for_rating, ->(rating) { where('rating_from <= ? AND rating_to >= ?', rating, rating) }
+  scope :with_scope, ->(scope) { where("#{scope}_active": true) } # need to use this?!
```

Comment:
I don't think you need to do this. The active flags indicate whether the rating will trigger something being visible on say the pupil or school dashboard or weekly emails. 

I don't think those contexts are relevant to recommendation, as if they've been assigned a rating and that rating has associated activities we should just suggest they do them. We use the same set of activities/interventions regardless of which scopes are active.

So just because we've decided to show something as a management priority and/or on the dashboards doesn't change what we recommend they do.

Reply:
Ahh that's great to know, thanks Leigh